cloud-common/src/main/java/com/sequenceiq/cloudbreak/cache/common/IdentityUserCache.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.cloudbreak.cache.common;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cache.CacheDefinition;
+
+import net.sf.ehcache.config.CacheConfiguration;
+
+@Service
```

Comment:
Why not Component? I dont see why its a service

Reply:
Because `@Service` is also a `@Component` and all other cache configurations are also `@Service`s. But you are right.

Reply:
I won't make big deal of it, I just don't like to see misused annotations